packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -485,3 +485,8 @@ export function isInstanceOfISnapshot(
 ): obj is ISnapshot {
 	return obj !== undefined && "snapshotFormatV" in obj && obj.snapshotFormatV === 1;
 }
+
+/**
+ * Default GroupId to ask for snapshot fetch when no groupId is specified.
+ */
+export const defaultGroupIdForSnapshot = "default";
```

Comment:
Why is this here? I think it either need to be inlined (and used in one place), or better - it should go to contracts.ts
That said, I expect it to be empty string, not "default". User could assign "default" to a groupID - this is too dangerous to have collision. I think our public FF APIs should assert that if groupId is provided, it's not empty, and use empty string to denote no groupId on the wire / protocol

Reply:
Removed it. Also, talking with server folks, to not have this in request.